controllers/clusterconfig_controller.go

```diff
@@ -226,6 +226,12 @@ func (r *ClusterConfigReconciler) writeInputData(ctx context.Context, config *re
 			return fmt.Errorf("failed to write pull secret: %w", err)
 		}
 
+		if config.Spec.ACMRegistration != nil {
+			if err := r.writeSecretToFile(ctx, &config.Spec.ACMRegistration.ACMSecret, filepath.Join(filesDir, "acm-secret.json")); err != nil {
```

Comment:
>Note that like the pull secret, the acm secret need to be added to the json/yaml in the ISO

@eranco74 that's what we're doing here.

Not sure if the comment was just for information or if you thought this part wasn't done yet.